releases.cfg

```diff
@@ -17,6 +17,7 @@ Releases
   {
     Modules = DIRAC, Web, VMDIRAC
     Externals = v6r6p3
+    DIRACOS = master
```

Comment:
Should we remove "Externals" at the same time?

Reply:
I do not know if the 'integration' is used somewhere.

Reply:
No, I think we want to keep it as a safeguard for one version